src/content/peerconnection/audio/css/main.css

```diff
@@ -17,6 +17,16 @@ button {
   width: 96px;
 }
 
+table {
+   border-collapse: collapse;
+}
+
+th, td {
+  border: 1px solid black;
+}
+
+tr:hover {background-color:#f5f5f5;}
```

Comment:
Put the property on a separate line like the rest.

Reply:
Done.